packages/arb-token-bridge-ui/tests/e2e/specs/approveToken.cy.ts

```diff
@@ -6,25 +6,14 @@ import {
 } from '../../support/common'
 
 const ERC20TokenAddressL1 = Cypress.env('ERC20_TOKEN_ADDRESS_L1')
+const ERC20Amount = '0.000000000001'
 
 describe('Approve token', () => {
   // log in to metamask
-  before(() => {
-    cy.login({ networkType: 'L1' })
-  })
-  after(() => {
-    // after all assertions are executed, logout and reset the account
-    cy.logout()
-  })
-  beforeEach(() => {
-    cy.restoreAppState()
-  })
-  afterEach(() => {
-    cy.saveAppState()
-  })
 
-  context('User approves and deposits ERC-20 token', () => {
-    it('should import token through UI', () => {
+  context('Approve token', () => {
```

Comment:
There's the same `Approve token` label in describe and context?

Reply:
removed it, it's not necessary